### PR TITLE
[WIP][Flow decomposition] Add parameters

### PR DESCRIPTION
**Pull request by OpenSuze**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x]Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
Internally

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Feature

**What is the current behavior?** *(You can also link to an open issue here)*
The flow decomposition parameters are only managed with the YAML file.

**What is the new behavior (if this is a feature change)?**
The flow decomposition parameters are also managed with the python interface.

**Does this PR introduce a breaking change or deprecate an API?** *If yes, check the following:*
- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:
Waiting for #470 
(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: data/NETWORK_LOOP_FLOW_WITH_COUNTRIES.uct [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+FLOAD 11 LOAD         0 0        100.000 0.00000 0.00000 0.00000
+##ZBE
+BGEN  11 GEN          0 2 400.00 0.00000 0.00000 -100.00 0.00000 1000.00 -1000.0 1000.00 -1000.0
+BLOAD 11 LOAD         0 0        100.000 0.00000 0.00000 0.00000
+##L
+EGEN  11 FGEN  11 1 0 1.0000 0.0500 0.000000    480 LINE
+FGEN  11 BGEN  11 1 0 1.0000 0.0500 0.000000    480 LINE
+BGEN  11 BLOAD 11 1 0 1.0000 0.0500 0.000000    480 LINE
+BLOAD 11 FLOAD 11 1 0 1.0000 0.0500 0.000000    480 LINE
+FLOAD 11 ELOAD 11 1 0 1.0000 0.0500 0.000000    480 LINE
```
Comment (EtienneLt) [inline data/NETWORK_LOOP_FLOW_WITH_COUNTRIES.uct]: git seems to need a new line at the end of the file

File: data/NETWORK_PST_FLOW_WITH_COUNTRIES.uct [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+##ZBE
+BLOAD 11              1 0 400.00 0.00000 0.00000 0.00000 0.00000 0000.00 -0000.0 0000.00 -0000.0
+BLOAD 12              0 2 400.00 200.000 0.00000 -100.00 0.00000 1000.00 -1000.0 1000.00 -1000.0
+##L
+FGEN  11 BLOAD 12 1 0 0.5000 1.5000 0.000000    480 LINE
+FGEN  11 BLOAD 11 1 0 1.0000 3.0000 0.000000    480 LINE
+##T
+BLOAD 11 BLOAD 12 2 0 400.0 400.0 1000. 0.5000 1.5000 150.0000 200.00   2000 PST
+##R
+BLOAD 11 BLOAD 12 2                    0.680 90.00 16   0       SYMM
```
Comment (EtienneLt) [inline data/NETWORK_PST_FLOW_WITH_COUNTRIES.uct]: same here

File: tests/test_flow_decomposition.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+            setattr(parameters, attribute, value)
+            assert value == getattr(parameters, attribute)
+            print(parameters)
+
+            df=pp.flowdecomposition.run(net, parameters)
+            print(df)
+
+if __name__ == "__main__":
+    test_flow_decomposition_parameters()
+    test_flow_decomposition_run_no_parameters()
```
Comment (EtienneLt) [inline tests/test_flow_decomposition.py]: same here

Comment (OpenSuze) [reply tests/test_flow_decomposition.py]: done

File: data/NETWORK_PST_FLOW_WITH_COUNTRIES.uct (hunk 2 above)
Comment (OpenSuze) [reply data/NETWORK_PST_FLOW_WITH_COUNTRIES.uct]: done

File: data/NETWORK_LOOP_FLOW_WITH_COUNTRIES.uct (hunk 1 above)
Comment (OpenSuze) [reply data/NETWORK_LOOP_FLOW_WITH_COUNTRIES.uct]: done

**Pull request closed without merging**